index.js

```diff
@@ -44,10 +44,22 @@ function getEmailHash(email) {
  * Generates random email in given domains
  * @param {Array} domains
  * @param {number} [len=7]
+ * @param {string} prefix
  * @returns {string}
  */
-function getRandomEmail(domains, len = 7) {
-  const name = Math.random().toString(36).substring(len);
+function getRandomEmail(domains, len, prefix) {
+  if(!len) { len = 7; }
+  if(!prefix) { prefix = ''; }
+  
+  const alfabet = '1234567890abcdefghijklmnopqrstuvwxyz';
+  
+  let name = !prefix ? '' : prefix + '-';
+  
+  for(let i = 0; i < len; i++) {
+    let randomChar = Math.round(Math.random() * (alfabet.length - 1));
```

Comment:
`randomChar` declared as `let` but never changed in block. `const` is more suitable in this case.